lib/index.ts

```diff
@@ -6,14 +6,19 @@ import * as _ from 'lodash';
 import {PkgTree, DepType, parseManifestFile,
   getDependencyTreeFromPackagesConfig, getDependencyTreeFromProjectJson,
   getDependencyTreeFromProjectFile, ProjectJsonManifest,
-  getTargetFrameworksFromProjectFile} from './parsers';
+  getTargetFrameworksFromProjectFile,
+  getTargetFrameworksFromProjectConfig} from './parsers';
 
-const PROJ_FILE_EXTENSION = [
+const PROJ_FILE_EXTENSIONS = [
   '.csproj',
   '.vbproj',
   '.fsproj',
 ];
 
+const PROJ_CONFIGS = [
```

Comment:
not sure we need this as an array the only file type supported by the parser for this is `packages.config`

Reply:
I did think that when I added it but if there's never going to be any other types of project configs files we can just hard code the check for packages.json. I'll change it.